components/virtual-node/virtual-node-rpcops-service-impl/src/main/kotlin/net/corda/virtualnode/rpcops/factories/RequestFactory.kt

```diff
@@ -0,0 +1,15 @@
+package net.corda.virtualnode.rpcops.factories
+
+import net.corda.data.virtualnode.VirtualNodeAsynchronousRequest
+import net.corda.libs.virtualnode.endpoints.v1.types.CreateVirtualNodeRequest
+import net.corda.virtualnode.HoldingIdentity
+
+interface RequestFactory {
```

Comment:
Just to highlight the fact that there is no intention to use it outside of this module:
```suggestion
internal interface RequestFactory {
```

Reply:
fixed